t/134-worker-FFI.t

```diff
@@ -0,0 +1,116 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
Please do not use capital letters like "FFI" in the test file name. Better use lower-case letters consistently, like `134-worker-ffi.t`.